Makefile

```diff
@@ -38,8 +38,8 @@ postgres: postgres-configure
 	+@echo "Compiling PostgreSQL"
 	$(MAKE) -C tmp_install/build MAKELEVEL=0 install
 	+@echo "Compiling contrib/zenith"
-	(cd vendor/postgres/contrib/zenith && \
-	$(MAKE) PG_CONFIG=$(abspath tmp_install)/bin/pg_config install USE_PGXS=1)
+	$(MAKE) -C tmp_install/build -f ../../vendor/postgres/contrib/zenith/Makefile \
+	PG_CONFIG=$(abspath tmp_install)/bin/pg_config install USE_PGXS=1 install
```

Comment:
How about this?
```
$(MAKE) -C tmp_install/build/contrib/zenith install
```
Maybe with a `MAKELEVEL=0`, although it seems to work without it, too.

Reply:
Yeah, that looks better.
Pushed new version.